config/host/config.exs

```diff
@@ -1,3 +1,33 @@
 import Config
 
+config :xebow,
+  leds: [
```

Comment:
Can we nest this one more level?

i.e.:

```elixir
config :xebow, layout: [leds: ..., keys: ...]
```

Reply:
Yes